OutOfSchool/OutOfSchool.BusinessLogic/Models/Workshops/WorkshopBaseDto.cs

```diff
@@ -55,21 +55,25 @@ public class WorkshopBaseDto : IValidatableObject
     [Range(0, 120, ErrorMessage = "Max age should be a number from 0 to 120")]
     public int MaxAge { get; set; }
 
-    [Required]
     [ModelBinder(BinderType = typeof(JsonModelBinder))]
+    [CollectionNotEmpty(ErrorMessage = "At least one DateTime range is required")]
     public List<DateTimeRangeDto> DateTimeRanges { get; set; }
 
+    [Required]
```

Comment:
does this attribute make sense for non nullable fields ?

Reply:
O.k. 
Removed [Required] attribute from the bool type properties - IsPaid, IsInclusive, IsSelfFinanced and IsSpecial of WorkshopBaseDto class.

Reply:
WorkshopBaseDto class already uses data annotation attributes (like [Required], [MinLength], [MaxLength], [Range]) etc. Also we use checking  ModelState.IsValid property in Create() action of WorkshopControllerso so  implementing IValidatableObject provides a single, unified validation mechanism.